test/e2e/framework/util.go

```diff
@@ -376,10 +376,10 @@ func SkipIfMissingResource(clientPool dynamic.ClientPool, gvr schema.GroupVersio
 }
 
 // ProvidersWithSSH are those providers where each node is accessible with SSH
-var ProvidersWithSSH = []string{"gce", "gke", "aws"}
+var ProvidersWithSSH = []string{"gce", "gke", "aws", "local"}
```

Comment:
fair. I'm pretty sure these lists are meant to match the switch statement you updated, and vagrant's switch case was added in #32560, though this list wasn't updated. @dshulyak can maybe update it.